build.cake

```diff
@@ -197,6 +203,12 @@ var BUG_TRACKER_URL = new Uri(GITHUB_SITE + "/issues");
 var DOCS_URL = new Uri(WIKI_PAGE);
 var MAILING_LIST_URL = new Uri("https://groups.google.com/forum/#!forum/nunit-discuss");
 
+// Nuspec-files don't handle forward slash in path in combination with recursive wildcards
+// https://github.com/cake-build/cake/issues/2367
+// https://github.com/NuGet/Home/issues/3584
+var TOOLS_SOURCE  = BIN_SRC + "**/" + OUTPUT_ASSEMBLY;
+TOOLS_SOURCE = TOOLS_SOURCE.Replace("/", @"\");
```

Comment:
I'm wondering if this is causing the failure to build on Linux? The message "Error: Unterminated string literal." occurs right when Cake is installing the tools. We may not want to do this substitution when running on Linux.

Another alternative: upgrade to the latest Cake build and see what happens.

I've had a lot of trouble getting my own .NET Core projects to build on Linux. In the end, I used some tailored bootstrap scripts to load the dotnet Cake tool rather than use Cake under mono. That's a bigger job however.

Reply:
Yes, I suspected the same. I will try having two nuspec-entries first (one pr target) instead of using globbing. I already tried the latest cake (0.38.4) and it didn't help.

Reply:
I tried updated to latest Cake in the result-writer repo, and it seemed to work better. Got as far as to know the Mono-version we build with don't support SDK-style csprojects. So need advise on how far up I should bump the mono-version.